packages/react-native-editor/src/setup.js

```diff
@@ -0,0 +1,140 @@
+/**
+ * Internal dependencies
+ */
+import initialHtml from './initial-html';
+import setupApiFetch from './api-fetch-setup';
```

Comment:
Not sure it matters, but I've generally noticed we list dependences in the order of:

1. External
2. WordPress
3. Internal

Should we relocate this block to be last?

Reply:
Thanks for spotting this, I reorder the imports in https://github.com/WordPress/gutenberg/pull/37073/commits/738938f41c06cddc5bbfbf9cd5543de585960526.